ui/layouts/root-layout.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { type ReactNode } from 'react';
+import { Header } from './header';
+
+export const RootLayout = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className="w-full h-full flex flex-col max-w-[clamp(576px,85vw,798px)] sm:mx-auto group-[.app--sidepanel]:max-w-[490px]">
```

Comment:
1. The fluid layout: `max-w-[clamp(576px,85vw,798px)]`
2. Sidepanel override: `max-w-[490px]`